### PR TITLE
Added test for EAP7 tour to NeedHelpTestCase

### PR DESCRIPTION
**Pull request by honza-kasik**

### CODE REVIEW

File: basic/src/test/java/org/jboss/hal/testsuite/test/homepage/ModuleTestCase.java [hunk 1]
```diff
@@ -81,7 +81,7 @@ public void configuration() throws Exception {
         Class <? extends BasePage> expectedAddressPage = ConfigUtils.isDomain() ?
                 DomainConfigEntryPoint.class
               : StandaloneConfigEntryPoint.class;
-        assertModule("config", expectedAddressPage, 1);
+        assertModule("config", expectedAddressPage, 2);
```
Comment (zebrik) [inline basic/src/test/java/org/jboss/hal/testsuite/test/homepage/ModuleTestCase.java:84]: Can you please extract this fix to separate commit?

File: basic/src/test/java/org/jboss/hal/testsuite/test/homepage/NeedHelpTestCase.java [hunk 2]
```diff
@@ -85,6 +91,21 @@ public void getHelpTest() throws Exception {
         assertLinksArePresent(section, links);
     }
 
+    @Test
+    public void newToEAP7Test() {
+        WebElement root = homePage.getContentRoot();
+
+        assertElementIsPresent("'New to EAP?' text should be present.", root,
+                new ByJQuery(":contains('" + PropUtils.get("homepage.newtoeap.text") + "')"));
```
Comment (zebrik) [inline basic/src/test/java/org/jboss/hal/testsuite/test/homepage/NeedHelpTestCase.java:99]: Please add related rows `homepage.newtoeap.text` and `homepage.newtoeap.link` to `common/src/main/resources/label.properties` as well ;-)

File: basic/src/test/java/org/jboss/hal/testsuite/test/homepage/ModuleTestCase.java (hunk 1 above)
Comment (honza-kasik) [inline basic/src/test/java/org/jboss/hal/testsuite/test/homepage/ModuleTestCase.java:84]: Extracted to separate commit

File: basic/src/test/java/org/jboss/hal/testsuite/test/homepage/NeedHelpTestCase.java (hunk 2 above)
Comment (honza-kasik) [inline basic/src/test/java/org/jboss/hal/testsuite/test/homepage/NeedHelpTestCase.java:99]: Added label.properties to commit :)

**Comment by honza-kasik** [comment]
Split original commit and added `label.properties` to changes.

**Comment by zebrik** [comment]
LGTM. Thank you.
